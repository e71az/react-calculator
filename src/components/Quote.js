import React from 'react';

const Quote = () => <h1>Quote</h1>;

export default Quote;
