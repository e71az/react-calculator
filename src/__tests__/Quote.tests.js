import React from 'react';
import renderer from 'react-test-renderer';
import MainNavbar from '../components/NavBar';
import Quote from '../components/Quote';

describe('Renders Quote component', () => {
  it('Should render the navbar and quote components correctly ', () => {
    const tree = renderer
      .create(
        <>
          <MainNavbar />
          <Quote />
        </>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
