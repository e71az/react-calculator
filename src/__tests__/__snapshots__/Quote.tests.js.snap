// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Quote component Should render the navbar and quote components correctly  1`] = `
Array [
  <nav
    className="navbar navbar-expand-lg navbar-dark bg-dark"
  >
    <a
      className="navbar-brand"
      href="#home"
    >
      Math Magicians
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        className="ml-auto navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rb-event-key="/"
          href="/"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          Home
        </a>
        <a
          className="nav-link"
          data-rb-event-key="/calc"
          href="/calc"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          Calculator
        </a>
        <a
          className="nav-link"
          data-rb-event-key="/quote"
          href="/quote"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          Quote
        </a>
      </div>
    </div>
  </nav>,
  <div
    className="home-container ml-5 mt-5"
  >
    <h1>
      Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
    </h1>
  </div>,
]
`;
